build.gradle

```diff
@@ -1,6 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    ext.kotlin_version = '1.3.31'
```

Comment:
This change is part of android Studio magic. Without this, I couldn't even run any test. Maybe it's because I'm not understanding 100%  why android studio add this line, but I think it's to manage different kotlin version (there was a difference between gradle and android studio I think). Any way, what should I do? Just by branching from master, I couldn't run test